sdk/deviceupdate/azure-deviceupdate/azure/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
I'm not sure if you manually added in this init file, but removing the `no-namespace-folders: true` from your [config file ](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/deviceupdate/data-plane/readme.python.md) will automatically generate this for you